test/packages/good/data_stream/k8s_data_stream/manifest.yml

```diff
@@ -1,6 +1,5 @@
 title: Data stream using Kubernetes service deployer for tests
 type: logs
-release: experimental
```

Comment:
I might be lost in the other email thread, but I assume we agreed on keeping the data stream release tag, right?

Reply:
Yes, I removed the deprecation (in https://github.com/elastic/package-spec/pull/256/commits/366c8d63d04b2a55a2d63d55ffec340d090ca292), but it is still optional. I wanted to be sure that we have test cases with good packages without the tag.